agrirouter-api-dotnet-standard-impl/Service/Messaging/Vcu/OnboardVcuService.cs

```diff
@@ -17,18 +17,15 @@ namespace Agrirouter.Impl.Service.Messaging.Vcu
     /// </summary>
     public class OnboardVcuService : IOnboardVcuService
     {
-        private readonly MessagingService _messagingService;
-        private readonly EncodeMessageService _encodeMessageService;
+        private readonly IMessagingService<MessagingParameters> _messagingService;
 
         /// <summary>
         /// Constructor.
         /// </summary>
         /// <param name="messagingService">-</param>
-        /// <param name="encodeMessageService">-</param>
-        public OnboardVcuService(MessagingService messagingService, EncodeMessageService encodeMessageService)
+        public OnboardVcuService(HttpMessagingService messagingService)
```

Comment:
Why are you using HttpMessagingService here; in all other classes it's the general IMessagingService